re/ReForm.re

```diff
@@ -84,7 +84,7 @@ module Create = (Config: Config) => {
              ~setSubmitting: ReasonReact.Callback.t(bool),
              ~setError: ReasonReact.Callback.t(option(string))
            ) =>
-           unit,
+           unit=ignore,
```

Comment:
I think this is pretty good ideas indeed. you can mark this as an optional argument by replacing `=ignore` => `=?` and handle the `option` type in the logic. `ignore` is the wrong way to do this

Reply:
exactly what @thangngoc89 said, in this case `ignore` does not work because it only placeholdes functions with the signature `'a => unit` and in this case it is a position argument and two labeleds ones, we could also just give a `(_, ~setSubmitting as _setSubmitting, ~setError as _setError) => ()`

Reply:
Awesome gonna try out and read more about this.